### DISCUSSION TITLE
I have some problems after running autoflash-7455.sh

### DISCUSSION THREAD

**Discussion by kyume1995** [danielewood/sierra-wireless-modems#172] [General]
![Capture](https://user-images.githubusercontent.com/14883677/169120309-731392ee-d2ed-4e1b-8b9a-d7e183e84f5c.PNG)
![dd4dc348-a3e5-46a4-882d-41b5e0f4d94c](https://user-images.githubusercontent.com/14883677/169120316-629cdbb1-7006-4ecd-96c2-26d5eaa71d1d.jpg)
![0efae3b8-2ce9-40b4-b946-3b39e8bc5ac7](https://user-images.githubusercontent.com/14883677/169120325-c332426f-4a07-4402-be58-bad379ab9009.jpg)
![5d6a0b57-fd33-4406-a4f1-64b2fcb4a269](https://user-images.githubusercontent.com/14883677/169120332-3c0d746d-4769-41f3-aaeb-29a88aa4af3a.jpg)
![dell_dw5811e_wwan_card_mr7vt_zoom_1](https://user-images.githubusercontent.com/14883677/169121554-f8109c14-f252-4807-87ba-4a16a6195437.png)

**Comment by thinhnguyenhcm** [comment]
I recommend you do some steps.
1. RUN AT
AT!ENTERCND="A710"
AT!NVRESTORE=0
Wait a minutes
AT!RESET

2. Download generic firmware Windows version and reflash: https://source.sierrawireless.com/resources/airprime/minicard/74xx/em_mc74xx-approved-fw-packages
